### PR TITLE
making sure this works

### PR DESCRIPTION
**Pull request by johngruen**
This is not for production, duh

### CODE REVIEW

File: app/styles/_variables.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@ $button-background-hover-color: #08c;
 $error-background-color: #d63920;
 $info-background-color: #0095dd;
 $error-text-color: #d63920;
-$html-background-color: #f2f2f2;
+$html-background-color: lime;
```
Comment (pdehaan) [inline app/styles/_variables.scss]: Should this be "salmon"?

**Comment by pdehaan** [comment]
@johngruen Did this auto-deploy test thing work for you, or did you need some help?

**Comment by pdehaan** [comment]
https://ux.dev.lcip.org/ver.json
```js
{
  version: "0.33.0",
  commit: "cb4651f7b2ac178d96875b22fc51d56b92bf102c",
  l10n: "d10f04b850",
  tosPp: "dd16fd3dbe"
}
```

Seems to be from 23 days ago: https://github.com/mozilla/fxa-content-server/commits/ux-experiments

@vladikoff to investigate

**Comment by vladikoff** [comment]
Investigation complete. Thanks to @jrgm the ux branch works!

**Comment by jrgm** [comment]
Yeah, so, in a rare event, the auto-update broke `yum` while updating `yum` which left `ansible` unable to complete an auto-update. I fixed `yum` and things are rolling again.

**Comment by pdehaan** [comment]
Great debug work, @vladikoff and @jrgm!
Also, excellent design Mr @johngruen. I totally love that lime green!

manage_account

**Pull request closed without merging**